content/events/international-rse-day/_index.md

```diff
@@ -17,13 +17,15 @@ The Nordic-RSE association is contributing to the International RSE day with an
 (all times in CEST)
 
 - All day, at anytime come and say hello in [https://coderefinery.zulipchat.com](https://coderefinery.zulipchat.com) in our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014). We do not have any registration form for this event but we would like to encourage our community to come along in zulip to say hello! 
-- 09:00 Introduce yourself (2 hours come along helpdesk session)
+- 09:00 Introduce yourself (2 hours come along helpdesk session in zulip chat)
   The goal of this session is to provide support to prepare a 1 min video about yourself:
      - your name, institution, RSE project you are working on e.g. what software are you contributing to (language, topic and whatever you can say in less than 1 minute!)
 
      All the videos will be published in our youtube channel and we will also make a "wall of fame" on the nordic-RSE website. 
 
-     Nordic-RSE staff will provide helpdesk so they can support you to prepare the video but of course, feel free to skip this session and send us directly the link to your 1 minute video!
+     Nordic-RSE staff will provide helpdesk in our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014) so they can support you to prepare the video but of course, feel free to skip this session and send us directly the link to your 1 minute video!
+     
+     If you want to an example of what you can have in your 1mn video, have a look [here](https://vimeo.com/629975390). Once your video is ready, share the link with us on our [nordic-rse/international RSE day Oct 14 stream](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse/topic/international.20RSE.20day.20Oct.2014).
```

Comment:
When I clicked the vimeo link it said "video not rated, log in to watch", is this intentional? (I never used vimeo)

Reply:
Weird. Could it depend on the browser? Even when using an incognito window, I don't get this message! Or maybe the video was not fully published yet and it is now?

Reply:
using incognito in chrome
![image](https://user-images.githubusercontent.com/49938764/136953481-cd809484-8e5f-42ce-be87-8ace751096e9.png)

Reply:
Ok. I got it. Is it better now? I did not know I had to se the audience manually! Thanks!.